pkgs/applications/networking/instant-messengers/signal-desktop/signal-desktop-darwin.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv, lib, fetchurl, _7zz }:
+stdenv.mkDerivation rec {
+  pname = "signal-desktop";
+  version = "7.28.0";
+
+  src = fetchurl {
+    url = "https://updates.signal.org/desktop/${pname}-mac-universal-${version}.dmg";
```

Comment:
Please don't re-use `pname`.

Using `pname` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21